avro/src/main/java/com/fasterxml/jackson/dataformat/avro/deser/ArrayReader.java

```diff
@@ -18,18 +21,18 @@
 
     protected String _currentName;
     
-    protected ArrayReader(AvroReadContext parent, AvroParserImpl parser)
+    protected ArrayReader(AvroReadContext parent, AvroParserImpl parser, Schema schema)
```

Comment:
I don't want Avro `Schema` type to be exposed at reader level.
Equivalent information may of course.